docs/Training-config.yml

```diff
@@ -0,0 +1,145 @@
+
+# Parameters for microDL virtual staining model training
+
+# The following configuration file defines the parameters to go from data processing to optimizing the microDL model.
+# Inorder to optimize the microDL model, which includes building the network and optimizing the weights, the input data provided is utilized.
+# Tiles of input data with different forms of noise and aberrations will accustom the model to work with future data which will contain basically similar
+# biological information but will be perturbed by different conditions (like change in image quality with differing imaging systems,
+# difference in sample condition, due to difference in cells, infections, drug treatments, cell, confluence, etc).
+# Initial conditions for initiating learning by the model is also provided, such as learning rate, step size, number of epochs, etc.
+
+# dataset parameters defining the different channels, how to split the dataset for training, validation and testing,
+# and image processing used for diversifying the data conditions, used for training the model and thus optimizing the weights.
+dataset:
+
+# Specify the directory in which your preprocessed tiles are stored
+  data_dir: '/home/Processed_temp_dir/tiles_256-256_step_128-128'
+
+# specify the model task, if it is regression model
+  model_task: regression
+
+# Defines the tile height
+  height: 256
+
+# specify the label-free channel to be used for predicting the stained image
+  input_channels: [2]
+
+# This is the extra channel which has been added to store the mask produced by segmenting the fluorescence stained image channel.
+  mask_channels: [3]    # This will be last input channel id + 1
+
+# define if preprocessing is performed?
+  preprocess: False
+
+# Define on what basis you want to split the input data folder for creating training image set, validation set and test set
+  split_by_column: pos_idx  # here 'pos_idx' indicates the data set was divided based on position list
+
+# Define the split ratio of data set
```

Comment:
Define what ratio of the data set will be split into testing, training and validation, respectively (test + train + val = 1).